cv19ResSupportV3.Tests/ConnectionString.cs

```diff
@@ -7,10 +7,10 @@ public static class ConnectionString
         public static string TestDatabase()
         {
             return $"Host={Environment.GetEnvironmentVariable("DB_HOST") ?? "127.0.0.1"};" +
-                   $"Port={Environment.GetEnvironmentVariable("DB_PORT") ?? "5432"};" +
+                   $"Port={Environment.GetEnvironmentVariable("DB_PORT") ?? "5433"};" +
```

Comment:
It's possible that your current build is working because your machine is keeping the old images around (or maybe because I'm wrong).
Try doing `docker system prune -a` and rebuild everything from scratch.

Reply:
If you do the prune thing, then you'll need to also stop all of the containers before that, otherwise, the docker images in use cannot be removed.

I think, `docker stop $(docker ps -aq)` should work.